src/aws/helpers/cfnStackExists.ts

```diff
@@ -0,0 +1,27 @@
+import CfnStackNotFound from '../../shared/errors/CfnStackNotFound.js';
+import getCfnStackExports from './getCfnStackExports.js';
+
+/**
+ * Confirm that a CloudFormation stack exists
+ * @author Jay Luker, Benedikt Arnarsson
+ * @param {string} stackName name of the stack to search for.
+ * @param {string} [profile='default'] AWS profile to use.
+ * @return {Promise<boolean>} whether the CloudFormation stack exists or not.
+ */
+const cfnStackExists = async (
+  stackName: string,
+  profile = 'default',
```

Comment:
Could this look for an $AWS_PROFILE env var before defaulting to 'default'? My default profile is called 'test'.